model_manager/src/transformers/BaseTransformer.py

```diff
@@ -2,21 +2,21 @@
 
 class BaseTransformer:
     @abstractmethod
-    def __init__(self, config):
+    def __init__(self, config: dict):
         """
-        config: dict passed from the pv_mappings.yaml files
+        config: dict passed from the pv_mappings.yaml files.
         """
         pass
     
     @abstractmethod
     def transform(self):
         """
-        Initiate transform function to transform the input data
+        Call transform function to transform the input data, see SimpleTransformer in model_manager/src/transformers/BaseTransformers.py for an example.
         """
         pass
     
     @abstractmethod
-    def handler(self, pv_name, value):
+    def handler(self, pv_name: str, value: dict | float | int):
```

Comment:
Add description of what the objects are in docstring:
* pv_name and the value associated with that PV
* this comes from me being ignorant but currently it cant handle arrays?

The way this method is implemented [SimpleTransformer](https://github.com/MatInGit/lume-deployment/blob/e7492b0eb35fec5535ecd535d063ff947545feaa/model_manager/src/transformers/BaseTransformers.py#L45C1-L66C1) class is confusing. They seem to be dictionaries and not primitive types.

Reply:
I think we are ok to merge, its been a while...